TaskList/app/models/task.rb

```diff
@@ -1,2 +1,6 @@
 class Task < ActiveRecord::Base
+	def self.completed
```

Comment:
Awesome use of a class method for finding all of the completed tasks! However, I don't see this method being used anywhere... perhaps in `TasksController#index` would be a good place?